### PR TITLE
General Removed max number of hints to show when entering wallets

### PR DESCRIPTION
**Pull request by standreinmcp**

### CODE REVIEW

File: src/screens/send/components/address-book/address-book.tsx [hunk 1]
```diff
@@ -43,8 +41,7 @@ export interface IExternalProps {
 
 export const mapStateToProps = (state: IReduxState, ownprops: IExternalProps) => {
     return {
-        contacts: selectContacts(state, ownprops.blockchain),
-        hints: state.app.hints
+        contacts: selectContacts(state, ownprops.blockchain)
```
Comment (georgeburduhos) [inline src/screens/send/components/address-book/address-book.tsx:44]: Why did you removed the hints from here?

File: src/screens/send/components/address-book/address-book.tsx [hunk 2]
```diff
@@ -22,16 +22,14 @@ import { Blockchain } from '../../../../core/blockchain/types';
 import Swipeable from 'react-native-gesture-handler/Swipeable';
 import { deleteContact, updateContactName } from '../../../../redux/contacts/actions';
 import { ICON_SIZE, normalize } from '../../../../styles/dimensions';
-import { IHints, HintsScreen, HintsComponent } from '../../../../redux/app/state';
+import { HintsScreen, HintsComponent } from '../../../../redux/app/state';
 import { updateDisplayedHint } from '../../../../redux/app/actions';
-import { DISPLAY_HINTS_TIMES } from '../../../../core/constants/app';
```
Comment (georgeburduhos) [inline src/screens/send/components/address-book/address-book.tsx:27]: remove DISPLAY_HINTS_TIMES if its not used in the app anymore

File: src/screens/send/components/address-book/address-book.tsx (hunk 1 above)
Comment (standreinmcp) [reply src/screens/send/components/address-book/address-book.tsx:44]: That was my understanding of the issue. "All the cards that have hidden menu..."

Comment (standreinmcp) [reply src/screens/send/components/address-book/address-book.tsx:44]: And the hints were not used anymore, after I deleted the check

**Comment by NovacAndrei** [comment]
@standreinmcp  You have to make sure that you add this functionality of opening hints on every Swipeable component. 
Search for `<Swipeable />` component in our code and add this functionality.

**Review by krisboit — APPROVED** [review]

**Review by georgeburduhos — APPROVED** [review]

**Pull request merged**